src/app/space/add-app-overlay/add-app-overlay.component.html

```diff
@@ -127,8 +127,15 @@ <h2 class="card-pf-title">
       <footer class="container-fluid">
         <div class="row">
           <div class="col-sm-12 code-imports--step_toolbar">
-            <button #cancelImportsButton id="cancelImportsButton" class="btn btn-primary" aria-label="Continue" [disabled]="selectedFlow === '' || projectName === '' || isProjectNameValid === false || isProjectNameAvailable === false || isProjectNameValid === undefined || isProjectNameAvailable === undefined"
-              (click)="routeToLaunchApp()">Continue</button>
+            <button #cancelImportsButton id="cancelImportsButton" class="btn btn-primary" aria-label="Continue"
+              [disabled]="selectedFlow === ''
+                || projectName === ''
+                || isProjectNameValid === false
+                || isProjectNameAvailable === false
+                || isProjectNameValid === undefined
+                || isProjectNameAvailable === undefined
```

Comment:
@vikram-raj can we encapsulate all `project` related checks to one meaningful property? This looks a bit hackish and could simply be `isProjectDefined`?

Reply:
@bartoszmajsak Yes, you are correct we can do it in the more simple way. I will do this code refactor in separate PR.